SPECS/aqc111u-module.spec

```diff
@@ -0,0 +1,65 @@
+%define vendor_name Marvell
+%define vendor_label marvell
+%define driver_name aqc111u
+
+%if %undefined module_dir
+%define module_dir extra
+%endif
+
+Summary: %{vendor_name} %{driver_name} device drivers
+Name: %{driver_name}-module
+Version: 1.3.3.0
+Release: 1%{?dist}
+License: GPL
+
+#Source taken from https://www.marvell.com/content/dam/marvell/en/drivers/AQ_USBDongle_LinuxDriver_1.3.3.0.zip
+Source0: %{driver_name}-%{version}.tar.gz
+
+Patch0: %{driver_name}-%{version}.patch
+
+BuildRequires: gcc
+BuildRequires: kernel-devel
+Provides: %{driver_name}-module
+Requires: kernel-uname-r = %{kernel_version}
+Requires(post): /usr/sbin/depmod
+Requires(postun): /usr/sbin/depmod
+
+%description
+%{vendor_name} %{driver_name} device drivers for the Linux Kernel
+version %{kernel_version}.
+
+%prep
+%setup -q -n %{driver_name}-%{version}
+
+%patch0 -p1
```

Comment:
TIP: you might prefer to use %autosetup (that automatically applies patches) in case you have more patches in the future.

Reply:
Thanks for the tip! Addressed in e4b1caa.